converter/output/rita/dbmanager.go

```
@@ -56,7 +54,7 @@ func NewOutputDB(ritaConf config.RITA) (OutputDB, error) {
 	db.dbRoot = ritaConf.GetDBRoot()
 	db.metaDBName = ritaConf.GetMetaDB()
 
-	db.ssn.DB(db.metaDBName).C(MetaDBDatabasesCollection).EnsureIndex(mgo.Index{
+	err = db.ssn.DB(db.metaDBName).C(constants.MetaDBDatabasesCollection).EnsureIndex(mgo.Index{
```

Found an error that was previously being swallowed and fixed it.